### DISCUSSION TITLE
GW2Efficiency-like account search and crafting calculator/to-do list

### DISCUSSION THREAD

**Discussion by Metallis** [blish-hud/Blish-HUD#678] [Module Ideas]
Account search is self explanatory.
I'm more interested in having a to-do list of the materials I need to gather with either a % like achievement tracker in game or absolute fractions and a bar. The gold cost, etc for actually crafting an item isn't _particularly_ needed live in game.

**Comment by Denrage** [comment]
Account search being implemented by tentose (Pentose on Discord):
https://github.com/tentose/Gw2ItemSearch

Overlaps therefor with https://github.com/blish-hud/Blish-HUD/discussions/639
